src/Access/AccessControlManager.cpp

```diff
@@ -269,12 +279,15 @@ void AccessControlManager::addStoragesFromUserDirectoriesConfig(
         String prefix = key + "." + key_in_user_directories;
 
         String type = key_in_user_directories;
-        if (size_t bracket_pos = type.find('['); bracket_pos != String::npos)
+        const size_t bracket_pos = type.find('[');
+        if (bracket_pos != String::npos)
```

Comment:
It seems you didn't change anything here?

Reply:
I use `bracket_pos` below, for preventing duplicate LDAP storages.

Reply:
Reverted, not relevant anymore.